src/lib/structures/game/StandardGame.ts

```diff
@@ -0,0 +1,152 @@
+import type { Message, Snowflake, User } from 'discord.js';
+import { Game, AcceptedAnswer, GameData, GameType } from '#game/Game';
+import { BrandingColors } from '#utils/constants';
+import { createEmbed } from '#utils/responses';
+import { container } from '@sapphire/framework';
+
+export class StandardGame extends Game {
+	public gameType: GameType.Standard = GameType.Standard;
+
+	public constructor(data: GameData) {
+		super(data);
+	}
+
+	public async guess(message: Message) {
+		const guess = message.content.toLowerCase();
+		const guessedBefore = this.guessedThisRound();
+		const guessedNow = this.processGuess(guess, message.author.id);
+		const promises: Promise<unknown>[] = [];
```

Comment:
guessedThisRound() must be called before processGuess() because processGuess() will update the state of `round`, and guessedBefore needs to get from the old state of `round`.

Reply:
No I mean the `promises` variable that doesn't get read or edited until a lot later down